quickstart/configure-mayastor.md

```diff
@@ -131,6 +134,7 @@ parameters:
   ioTimeout: "30"
   protocol: nvmf
   repl: "1"
+  thin: true
```

Comment:
given this is the first release of thin-provisioning we probably shouldn't make it the default until it's more battle tested? @avishnu

Reply:
okay, so I'll be removing it from the YAML example. But so that the users know that we have this functionality I'll put a note with the link to the 'Storage Class parameters page'.